generator.go

```diff
@@ -360,7 +373,7 @@ func (g *Gen) NewV7() (UUID, error) {
 //	When useUnixTSMs is false, it uses the Coordinated Universal Time (UTC) as a count of 100-
 //
 // nanosecond intervals since 00:00:00.00, 15 October 1582 (the date of Gregorian reform to the Christian calendar).
-func (g *Gen) getClockSequence(useUnixTSMs bool) (uint64, uint16, error) {
+func (g *Gen) getClockSequence(useUnixTSMs bool, atTime time.Time) (uint64, uint16, error) {
```

Comment:
nit: The doc string for this method needs to be updated to reflect the new behavior

Unrelated to this PR, but it seems like the formatting of the comment is off.  I don't think there's supposed to be a paragraph break between "100-" and  "nanosecond".